dashboard-ui/src/components/Research/tables/CalibrationData.jsx

```diff
@@ -0,0 +1,465 @@
+import React from "react";
+import '../../SurveyResults/resultsTable.css';
+import { useQuery } from 'react-apollo'
+import gql from "graphql-tag";
+import { RQDefinitionTable } from "../variables/rq-variables";
+import CloseIcon from '@material-ui/icons/Close';
+import { Autocomplete, Modal, TextField } from "@mui/material";
+import ph1DefinitionXLFile from '../variables/Variable Definitions RQ134_PH1.xlsx';
```

Comment:
Probably need a new version of this file, since it has things like the DRE alignment columns and `TA1_Name`, but lacks the `Calibration Alignment Score (Delegator|Observed_ADM (target))` column.